generators/utils.js

```diff
@@ -344,3 +370,457 @@ function buildEnumInfo(field, angularAppName, packageName) {
 function copyObjectProps(toObj, fromObj) {
     Object.assign(toObj, fromObj);
 }
+
+
+/**
+ * get a property by path in a array
+ *
+ * @param {array} array - array in which we want to search
+ * @param {string} name - property name to search "format myProperty.level2.level3"
+ * @param {string} generator - The generator
+ * @return {string} the value of property or undefined
+ */
+function getPropertyInArray(array, name, generator) {
+    return _.get(array, name);
+}
+
+/**
+ * Update or create a property by path in a array
+ *
+ * @param {array} array - array in which we want to update
+ * @param {string} name - property name to search "format myProperty.level2.level3"
+ * @param {string} generator - The generator
+ * @param {string} value - (optional) value of the property
+ */
+function updatePropertyInArray(array, name, generator, value) {
+    _.set(array, name, value);
+}
+
+/**
+ * Delete a property by path in a array
+ *
+ * @param {array} array - array in which we want to delete
+ * @param {string} name - property name to search "format myProperty.level2.level3"
+ * @param {string} generator - The generator
+ */
+function deletePropertyInArray(array, name, generator) {
+    _.unset(array, name);
+}
+
+/**
+ * Get a yaml property by path in a file
+ *
+ * @param {string} file - yaml file to search
+ * @param {string} name - property name to search "format myProperty.level2.level3"
+ * @param {string} generator - The generator
+ * @return {string} the value of property or undefined
+ */
+function getYamlProperty(file, name, generator) {
+    const treeData = jsyaml.safeLoad(generator.fs.read(file));
```

Comment:
This is not efficient if reading multiple props. Ideally, you should just load file once and read all required values from that. Its better to do this directly using the library. I dont see added value in having this method